person.rb

```diff
@@ -0,0 +1,25 @@
+# Person Class
+class Person
+  def initialize(age,
+                 name = 'Unknown',
+                 _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
```

Comment:
- [ ] In the definition of the parameters, I would recommend you remove the preceding underscore (`_`) from `_parent_permission: true` so as to avoid any obstruction when creating a **Person** object.

  ![temitope-underscore](https://user-images.githubusercontent.com/36542986/157083620-fe1c44f7-d52d-44d7-8f48-62045f3cc288.jpg)

- [ ] Your **initialize** method is missing the instance variable of `@parent_permission`. Kindly add that
  ```
  @parent_permission = parent_permission
  ```